go.mod

```diff
@@ -58,6 +59,25 @@ require (
 	gotest.tools/v3 v3.4.0
 )
 
+require (
```

Comment:
Could you add the commen lines about why this section is separated?

Reply:
oops I did not notice that, there is no requirement to separate them, I would merge them into one seciton.